CONTRIBUTING.md

```diff
@@ -21,6 +21,14 @@ or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any addi
 ## Recommended Tools
 * Visual Studio and Visual Studio Code https://visualstudio.microsoft.com/
 
+## Git
+
+### Conventions
+TODO
+
+### Branching Workflow
```

Comment:
So far, our branching workflow generally consists of using L1 (level 1) topic branches. (where the L0 branch is the `main` branch).

Reply:
This might be ignorance on my part, but I'm not aware of wide circulation of that terminology (L0, L1) outside of Microsoft. I think you could just say "topic branch" and link to the [Git documentation](https://git-scm.com/book/en/v2/Git-Branching-Branching-Workflows#_topic_branch).